### PR TITLE
Migrate features/adblocker page to Fluent (#9146) [skip l10n]

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
http://localhost:8000/en-US/firefox/features/adblocker/

## Issue / Bugzilla link
#9146

## Testing
```
./manage.py l10n_update
```

### CODE REVIEW

**Comment by peiying2** [comment]
Missed this one: `{ -brand-name-send } a Do Not Track signal`. We need a new PR for this one.

File: l10n/en/firefox/features/adblocker.ftl [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+features-adblocker-cover-your-trail-block = Cover your trail, block trackers
+features-adblocker-click-on-the-trackers = Click on the Trackers box and you’ll be able to block trackers in two ways. One way to block trackers is to do it when you’re working in a Private Window. Another way to do it is to block trackers in all windows. Keep in mind that if you choose to always block trackers, some pages might not load correctly.
+features-adblocker-take-a-bite-out-of-cookies = Take a bite out of cookies
+
+# Variables:
+#   $url (url) - link to https://support.mozilla.org/kb/storage
+features-adblocker-cookies-are-sent-by-websites = <a href="{ $url }">Cookies</a> are sent by websites you visit. They live on your computer and monitor what you’ve been doing on a site. When an airline hikes your rates because you’ve looked at plane tickets once that day, that is the handiwork of a cookie.
+
+features-adblocker-in-firefox-you-can-block = In { -brand-name-firefox }, you can block all third-party cookies or just those set by trackers. Be aware that blocking all cookies can break some sites.
+features-adblocker-send-a-do-not-track-signal = { -brand-name-send } a Do Not Track signal
```
Comment (peiying2) [inline l10n/en/firefox/features/adblocker.ftl:61]: The `Send` here is a verb. I missed this one. Need a new PR.

Comment (alexgibson) [reply l10n/en/firefox/features/adblocker.ftl:61]: Opened https://github.com/mozilla-l10n/www-l10n/pull/112 thanks

**Review by craigcook — APPROVED** [review]
🛡️

**Pull request merged**